components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -52,38 +44,18 @@ export default function () {
     const showStartWSModal = () => setIsTemplateModelOpen(true);
     const hideStartWSModal = () => setIsTemplateModelOpen(false);
 
-    const getRecentSuggestions: () => WsStartEntry[] = () => {
-        if (workspaceModel) {
-            const all = workspaceModel.getAllFetchedWorkspaces();
-            if (all && all.size > 0) {
-                const index = new Map<string, WsStartEntry & { lastUse: string }>();
-                for (const ws of Array.from(all.values())) {
-                    const repoUrl = Workspace.getFullRepositoryUrl(ws.workspace);
-                    if (repoUrl) {
-                        const lastUse = WorkspaceInfo.lastActiveISODate(ws);
-                        let entry = index.get(repoUrl);
-                        if (!entry) {
-                            entry = {
-                                title: Workspace.getFullRepositoryName(ws.workspace) || repoUrl,
-                                description: repoUrl,
-                                startUrl: gitpodHostUrl.withContext(repoUrl).toString(),
-                                lastUse,
-                            };
-                            index.set(repoUrl, entry);
-                        } else {
-                            if (entry.lastUse.localeCompare(lastUse) < 0) {
-                                entry.lastUse = lastUse;
-                            }
-                        }
-                    }
-                }
-                const list = Array.from(index.values());
-                list.sort((a, b) => b.lastUse.localeCompare(a.lastUse));
-                return list;
+    useEffect(() => {
+        const onKeyDown = (event: KeyboardEvent) => {
```

Comment:
**thought**: Somehow I was expecting to be able to trigger this modal from every page in the dashboard but opening this only while in the workspaces list sounds also of for the first iteration. ➿

Reply:
Is it a lot of work making it work from other Dashboard pages? I think it makes a big qualitative difference to the feature.

Reply:
Thanks for the reminder! It's not a lot of work. I'll do it in the next iteration 👍